notebooks/process_demetr_results.ipynb

```diff
@@ -0,0 +1,3162 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    cat_to_rev = all_res.loc[all_res['category'] == 35]
</pre><p>35 is hardcoded in a few places.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/ARC-m4st/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>